databricks/koalas/series.py

```diff
@@ -4343,10 +4343,12 @@ def replace(self, to_replace=None, value=None, regex=False) -> "Series":
         """
         if to_replace is None:
             return self.fillna(method="ffill")
-        if not isinstance(to_replace, (str, list, dict, int, float)):
-            raise ValueError("'to_replace' should be one of str, list, dict, int, float")
+        if not isinstance(to_replace, (str, list, tuple, dict, int, float)):
```

Comment:
@itholic can you also update the parameters in the docs?

Reply:
Sure, thanks for the comment!